packages/base-ui/src/services/dialog/dialog.service.ts

```diff
@@ -0,0 +1,14 @@
+import { createIdentifier, IDisposable } from '@wendellhu/redi';
+import { type Subject } from 'rxjs';
+
+import { type IDialogPartMethodOptions } from '../../views/components/dialog-part/interface';
+
+export const IDialogService = createIdentifier<IDialogService>('univer.dialog-service');
+
+export interface IDialogService {
+    open(params: IDialogPartMethodOptions): IDisposable;
+
+    close(id: string): void;
```

Comment:
Some JSDoc would be nice.

Reply:
The JSDoc may not fully describe the usage of this method, so I will provide a comprehensive usage example later :octocat: